extensions/default/src/Toolbar/ToolbarLayoutSelector.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useState, useCallback } from 'react';
 import PropTypes from 'prop-types';
-import { LayoutSelector as OHIFLayoutSelector, ToolbarButton } from '@ohif/ui';
+import { LayoutSelector as OHIFLayoutSelector, ToolbarButton, LayoutPreset } from '@ohif/ui';
 import { ServicesManager } from '@ohif/core';
 
 function ToolbarLayoutSelectorWithServices({ servicesManager, ...props }) {
```

Comment:
Please have a copy of the old one as legacyLayoutSelector if soemone is intereseted